Update Troubleshooting section in Code Origins doc (#29438)
DataDog/documentation: content/en/tracing/code_origins/_index.md

- To search for all spans that include Code Origins, use the query `@_dd.code_origin.type:*` in the [APM Trace Explorer][1].

> ### Code preview is not visible, or file not found

Reviewer: ```suggestion
### Code preview is not visible or the file is not found
```

> - Ensure that all [Source Code Integration][7] setup requirements are met, and that your `DD_GIT_*` tags are configured with the correct values.

Reviewer 2: I'm not sure about this change:
1. We're linking to the SCI page, which describes how to configure `DD_GIT_*`, but then we also mention it inline here with an "and", implying that it's something the user needs to do _besides_ what they are instructed to do in the SCI setup requirements. If we want to include this, we at least should replace "and" with something like "specifically" or "including".
2. If we decide to include it, I don't like that we call it "`DD_GIT_*` tags". IMO it should either be "`DD_GIT_*` environment variables", or it should be something like "Git information is embedded correctly in your build artifacts" to make it more generic like it is in the SCI setup requirements section.

> - Ensure that all [Source Code Integration][7] setup requirements are met, and that your `DD_GIT_*` tags are configured with the correct values.

Author: ```suggestion
- Ensure all [Source Code Integration][7] setup requirements are met, including your `DD_GIT_*` environment variables are configured with the correct values.
```

> - For transpiled Node.js applications (for example, TypeScript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews do not work. See the Node.js [Source Code Integration][9] documentation for more details.

Author: ```suggestion
- For transpiled Node.js applications (for example, TypeScript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews will not work. See the Node.js [Source Code Integration][9] documentation for more details.
```

> - For transpiled Node.js applications (for example, TypeScript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews will not work. See the Node.js [Source Code Integration][9] documentation for more details.

Reviewer: ```suggestion
- For transpiled Node.js applications (for example, TypeScript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews will not work. See the Node.js [Source Code Integration][9] documentation for more details.
```

> - Code Origins is designed to reference user code only, but in some cases, third-party code references may slip through. You can help us improve by reporting these cases support@datadoghq.com!

Author: ```suggestion
- Code Origins is designed to reference user code only, but in some cases, third-party code references may slip through. Please help us improve by reporting these cases to [Datadog support][13].
```

> - Code Origins is designed to reference user code only, but in some cases, third-party code references may slip through. Please help us improve by reporting these cases to [Datadog support][13].

Reviewer: ```suggestion
- Code Origins is designed to reference user code only, but in some cases, third-party code references may slip through. You can report these cases to [Datadog support][13] and help improve these references.
```
